### DISCUSSION TITLE
Will the styles of unused components be removed during compilation?

### DISCUSSION THREAD

**Discussion by euwei7** [jd-solanki/anu#198] [Announcements]
I started experimenting with anu, and in the documentation guide, I noticed that the `uno.config.ts` configuration file requires the addition of: `include: [/.*\/anu-vue\.js(.*)?$/, './**/*.vue', './**/*.md']`. This will enable unocss to scan the `anu-vue.js` file.

If anu compiles all components into the `anu-vue.js` file, will unocss generate styles for all components (including unused ones) in the CSS file when scanning the anu-vue.js file?

**Comment by jd-solanki** [comment]
No, It won't generate the styles of unused components or class. Let me know if there's any issue.
